package.json

```diff
@@ -23,11 +23,15 @@
     "@angular/platform-browser": "~4.0.3",
     "@angular/platform-browser-dynamic": "~4.0.3",
     "core-js": "~2.4.1",
+    "ng-redux": "^3.5.2",
+    "redux-devtools-extension": "^2.13.2",
```

Comment:
No redux dependency?

Reply:
> No redux dependency?

`redux` is pulled in through `ng-redux` as transitive dependency, [see here](https://github.com/angular-redux/ng-redux/blob/3.5.2/package.json#L60).

`ng-redux@^3.5.2` relies on `redux@^3.7.2` and let's say we add dependency on `redux@^4.x` this might break `ng-redux` expecting version 3.x APIs. This is why I didn't include `redux` dependency here.